project1/index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Cranes wife</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Abel|IM+Fell+French+Canon+SC|Notable&display=swap"
+        rel="stylesheet">
+
+</head>
+
+<body>
+    <hr />
+    <h1>Crane Wife</h1>
+    <hr />
+    <h2><a href="https://cjhauser.com/>"><em>By CJ Hauser</em></a></h2>
```

Comment:
1. You have an extra `>` here:
```suggestion
    <h2><a href="https://cjhauser.com/"><em>By CJ Hauser</em></a></h2>
```
2. I would not recommend using the `em` element here -- the byline "By CJ Hauser" is already an `h2`, so it has significance, and wouldn't be said aloud in a way that has stress emphasis, which is the purpose of the `em` element.